manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -191,11 +197,24 @@ public function registerContainerConfiguration(LoaderInterface $loader): void
             $loader->load($configDir.'/parameters.yml');
         }
 
+        if (file_exists($legacyDir.'/parameters.yml')) {
+            // Deprecation warning is already triggered above
+            $loader->load($legacyDir.'/parameters.yml');
+        }
+
         if (file_exists($configDir.'/config_'.$this->getEnvironment().'.yml')) {
             $loader->load($configDir.'/config_'.$this->getEnvironment().'.yml');
         } elseif (file_exists($configDir.'/config.yml')) {
             $loader->load($configDir.'/config.yml');
         }
+
+        if (file_exists($legacyDir.'/config_'.$this->getEnvironment().'.yml')) {
+            @trigger_error('Placing a config_'.$this->getEnvironment().'.yml in /app/config is deprecated since Contao 4.8. Place it in the root /config folder instead.', E_USER_DEPRECATED);
+            $loader->load($legacyDir.'/config_'.$this->getEnvironment().'.yml');
```

Comment:
Why are you loading the legacy files last? Shouldn't the new files be loaded last?

Reply:
Changed in 5a975321952fbcfc31a635829ca6f85009d0d505.